### ISSUE TITLE
jdk11 aarch64 build failure: Immediate value is out of range for subsimm

### ISSUE THREAD

**Issue by andrew-m-leonard** [eclipse/openj9#9796]
https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-linux-aarch64-openj9/353/console

```
01:18:11  * For target jdk__packages_attribute.done:
01:18:11  Assertion failed at ../omr/compiler/aarch64/codegen/GenerateInstructions.cpp:362: constantIsUnsignedImm12(imm)
01:18:11  VMState: 0x0005ff04
01:18:11  	Immediate value is out of range for subsimm
01:18:11  compiling java/lang/String.decompressedArrayCopy([BI[BII)V at level: warm
01:18:11  #0: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x6eabbc) [0x10026faabbc]
01:18:11  #1: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x6f5ba0) [0x10026fb5ba0]
01:18:11  #2: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x3f6b5c) [0x10026cb6b5c]
01:18:11  #3: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x3f84c8) [0x10026cb84c8]
01:18:11  #4: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x72aa44) [0x10026feaa44]
01:18:11  #5: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x2415c0) [0x10026b015c0]
01:18:11  #6: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x37ebf0) [0x10026c3ebf0]
01:18:11  #7: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0xe8988) [0x100269a8988]
01:18:11  #8: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x38bc98) [0x10026c4bc98]
01:18:11  #9: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x38a598) [0x10026c4a598]
01:18:11  #10: /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-aarch64-openj9/workspace/build/src/build/linux-aarch64-normal-server-release/jdk/lib/compressedrefs/libj9jit29.so(+0x3878d4) [0x10026c478d4]
```

**Comment by 0xdaryl** [comment]
@knn-k

**Comment by knn-k** [comment]
This is already fixed.
This happened because #9751 was merged before eclipse/omr#5263 got promoted.

**Comment by pshipton** [comment]
If it's already fixed, I'll go ahead and close it. Pls reopen or comment if it's still a problem.
